test/content-length.test.js

```diff
@@ -180,10 +174,9 @@ test('#2543 - wrong content-length with errorHandler', t => {
     method: 'GET',
     path: '/'
   })
-    .then(res => {
-      t.equal(res.statusCode, 500)
-      t.equal(res.headers['content-length'], '' + res.rawPayload.length)
-      t.same(JSON.parse(res.payload), { message: 'longer than 2 bytes' })
-      t.end()
+    .then(response => {
```

Comment:
let me guess, inject would throw?

Reply:
made another change see commit here: https://github.com/fastify/fastify/pull/5878/commits/febdf6f169769dea754dcf649d05ebdf17fd355d

but it still fails